net/dpdk/Makefile

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME := dpdk
+PKG_VERSION := 21.05
+
+PKG_SOURCE := dpdk-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL := http://static.dpdk.org/rel
+PKG_HASH := 1e12499b4c5fcdb57c83e5fe184ea6becddf7cf092893c2c5ef2efb0eec12f0b
+
+PKG_BUILD_DEPENDS:= meson/host ninja/host
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
+include $(TOPDIR)/feeds/packages/devel/meson/meson.mk
+include $(TOPDIR)/feeds/packages/devel/ninja/ninja.mk
```

Comment:
This line lets the build fail.
There is no ninja.mk anymore, see:
https://github.com/openwrt/packages/commit/f08eb8552c6bd71124a5407fb37abe6dc4982ac3

Reply:
@PolynomialDivision ..Thanks for Suggestion. I'll make necessary changes.